.github/workflows/update-pgxn-version.yml

```diff
@@ -16,21 +15,27 @@ jobs:
     name: Update pgxn configuration
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out the repo
+        uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone Tools branch
-        run: git clone --depth 1 --branch v0.8.1 https://github.com/citusdata/tools.git
+        run: git clone --depth 1 --branch update_properties_fixes https://github.com/citusdata/tools.git
```

Comment:
After merging https://github.com/citusdata/tools/pull/183, I will tag tools and will change here